### ISSUE TITLE
Orbot/Proxy support

### ISSUE THREAD

**Issue by JasperWeiss** [slapperwan/gh4a#302]
I'd be nice to have support for Orbot so that censorship and internet surveillance can be bypassed

**Comment by maniac103** [comment]
Wouldn't you want something like that more globally, that is, system wide?

**Comment by JasperWeiss** [comment]
Nope, that would include Google play services, whatsapp etc. Which kind of de-anonymizes the user.
